db/migrate/20200303023220_create_bookings.rb

```diff
@@ -0,0 +1,17 @@
+class CreateBookings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :bookings do |t|
+      t.date :date
+      t.time :start_time
+      t.integer :duration
+      t.float :total_fee
+      t.boolean :confirmed
```

Comment:
make the default false for confirmed boolean?

Reply:
Added the default with Guido's help